client/packages/invoices/src/OutboundShipment/DetailView/DetailView.tsx

```diff
@@ -35,6 +37,10 @@ export const DetailView: FC = () => {
     },
     [toItemRow, onOpen]
   );
+  const onAddItem = (draft?: Draft) => {
+    onOpen(draft);
+    setMode(ModalMode.Create);
```

Comment:
Is it ok for this modal to display `add-item` even thought it may not be a new item and not being able to move to next item ?

https://github.com/openmsupply/open-msupply/blob/afdab0916ed5ee213ec6c5a045b02b0a60f348c0/client/packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/OutboundLineEdit.tsx#L161

https://github.com/openmsupply/open-msupply/blob/afdab0916ed5ee213ec6c5a045b02b0a60f348c0/client/packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/OutboundLineEdit.tsx#L138

I think another good place to outline what the desired behaviour should be with test use case and expected behaviour

Reply:
If only we had a full set of requirements which detailed what the behaviour should be.. 🤔

Reply:
and yes, the enum check should be exhaustive and should only be showing add-item when the ModalMode is Create